src/aosm/azext_aosm/_configuration.py

```diff
@@ -126,19 +127,28 @@ def validate(self) -> None:
     def sa_manifest_name(self) -> str:
         """Return the Storage account manifest name from the NFD name."""
         return f"{self.nf_name}-sa-manifest-{self.version.replace('.', '-')}"
+
     @property
     def build_output_folder_name(self) -> str:
         """Return the local folder for generating the bicep template to."""
         arm_template_path = self.arm_template.file_path
         return (
-            f"{VNF_DEFINITION_OUTPUT_BICEP_PREFIX}{Path(str(arm_template_path)).stem}"
+            f"{DEFINITION_OUTPUT_BICEP_PREFIX}{Path(str(arm_template_path)).stem}"
         )
 
+
 @dataclass
 class HelmPackageConfig:
     name: str = "Name of the Helm package"
-    path_to_chart: str = "Path to the Helm chart"
-    depends_on: List[str] = field(default_factory=lambda: ["Names of the Helm packages this package depends on"])
+    path_to_chart: str = (
+        "File path of Helm Chart on local disk. Accepts .tgz, .tar or .tar.gz"
+    )
+    depends_on: List[str] = field(
+        default_factory=lambda: [
+            "Names of the Helm packages this package depends on. Leave as an empty string if no dependencies"
```

Comment:
It isn't an empty string, it is an empty list, if not dependencies.  
i.e. from your example config
            "depends_on": [
                
            ]